proto/viam/app/v1/app.proto

```diff
@@ -75,6 +75,18 @@ service AppService {
 
   rpc OrganizationGetSupportEmail(OrganizationGetSupportEmailRequest) returns (OrganizationGetSupportEmailResponse);
 
+  rpc OrganizationSetLogo(OrganizationSetLogoRequest) returns (OrganizationSetLogoResponse);
+  rpc OrganizationGetLogo(OrganizationGetLogoRequest) returns (OrganizationGetLogoResponse);
+
+  rpc EnableAuthService(EnableAuthServiceRequest) returns (EnableAuthServiceResponse);
+  rpc DisableAuthService(DisableAuthServiceRequest) returns (DisableAuthServiceResponse);
+
+  rpc CreateOAuthApp(CreateOAuthAppRequest) returns (CreateOAuthAppResponse);
+  rpc ReadOAuthApp(ReadOAuthAppRequest) returns (ReadOAuthAppResponse);
+  rpc UpdateOAuthApp(UpdateOAuthAppRequest) returns (UpdateOAuthAppResponse);
+  rpc DeleteOAuthApp(DeleteOAuthAppRequest) returns (DeleteOAuthAppResponse);
```

Comment:
fwiw we explicitly decided _not_ to implement delete in the original scope of work. this is because delete is a non-reversible and pretty large impact operation so we dont want anyone doing it by mistake. i know you want to implement all CRUD operations but 🤷‍♀️ . i dont think having to reach out to support to delete an app is an antipattern.

Reply:
My opinion is anyone who is savvy enough to be using a CLI to generate OAuth apps that work have sufficient knowledge to be deleting them ... maybe we can implement a two step safety solution where the user has to double confirm that they want to delete?

Reply:
its not about being savvy enough, its more that its a big operation that we dont want people to do lightly. agreed that in the CLI could be good to implement that prompt to confirm.

Reply:
I am pretty heavily leaning towards the ability to delete along with adding a double confirm step in the CLI that makes it super clear to the user that its a dangerous operation ... the change was agreed upon in scope too and (I think) I made it clear that it was a new command (i.e. not from your existing work)